assets/base.css

```diff
@@ -1868,6 +1868,7 @@ input[type='checkbox'] {
 
 /* component-quantity */
 .quantity {
+  background: rgb(var(--color-background));
   color: rgba(var(--color-foreground));
```

Comment:
Good reminder. I forget about that issue because it varies based on how much radius is applied. Waiting for confirmation that my assumption is correct about the inputs having opaque backgrounds, but I'll fix the radius issue regardless.

Reply:
Sounds good. I will re-review then 👍